src/main/java/org/openjdk/jextract/impl/ClassSourceBuilder.java

```diff
@@ -126,12 +126,16 @@ final void decrAlign() {
         sb.decrAlign();
     }
 
+    String format(String s, String... args) {
```

Comment:
I added this method here, then I use it in both `appendLines` and `appendIndentedLines`